cas-server-webapp/src/main/webapp/WEB-INF/cas.properties

```diff
@@ -20,11 +20,8 @@
 server.name=http://localhost:8080
 server.prefix=${server.name}/cas
 
-# Spring Security's EL-based access rules for the /status URI of CAS that exposes health check information
-# cas.securityContext.status.access=hasIpAddress('127.0.0.1')
-
-# Spring Security's EL-based access rules for the /statistics URI of CAS that exposes stats about the CAS server
-# cas.securityContext.statistics.access=hasIpAddress('127.0.0.1')
+# security configuration based on IP address to access the status and statistics pages
```

Comment:
I admit that pac4j seems more limited than Spring Security here as there is no EL syntax like `hasIpAddress`. Nonetheless, if ever you change this expression by: `hasRole`, you likely need to change your `securityContext.xml` configuration to compute this role. Like for pac4j.